explorer/file_test.cpp

```diff
@@ -2,45 +2,20 @@
 // Exceptions. See /LICENSE for license information.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+#include "absl/flags/flag.h"
```

Comment:
We seem to have a convention of using angle-bracket includes for third-party code other than llvm (see e.g. our usage of gtest). Should we follow that here?

Reply:
I think the convention should be to follow documented include usage since that's what we should expect systems to support.

The intent is to follow system documentation on how to include, for example `#include "absl/strings/str_join.h"` per https://abseil.io/docs/cpp/quickstart#creating-and-running-a-binary

For bazel libraries, we have `#include "tools/cpp/runfiles/runfiles.h"` echoing the above style.

For protobuf libraries, we do angle brackets, but you can see this style in https://protobuf.dev/reference/cpp/arenas/ so we are following the documentation.

I see gtest also suggests quote includes like `#include "gtest/gtest.h"` in https://google.github.io/googletest/primer.html#writing-the-main-function. I don't know why we're using angle brackets, it may have been a mistake; I believe this should be migrated, not all the others.